### PR TITLE
Update losses.py

### PR DESCRIPTION
**Pull request by mfmezger**
I'm not sure I missed anything, but I think it should be logits and not probas. Greetings

### CODE REVIEW

**Comment by kevinzakka** [comment]
Thanks for spotting, I had fixed dice but since I never used jaccard and tversky, forgot to update them as well.

**Review by kevinzakka — COMMENTED** [review]
Is the cast necessary?

**Comment by mfmezger** [comment]
I think the cast is necessary, but i am not sure. I will check that tomorrow. But i think the cast could fix the problem of Issue #6 .

**Comment by mfmezger** [comment]
Hi, so i checked it again, for my data its only working when i add the cast. Is this for your data not the case?

**Comment by mfmezger** [comment]
I removed the casts.

**Comment by kevinzakka** [comment]
Thanks @mfmezger!
